controllers/testrun_controller.go

```diff
@@ -268,7 +268,7 @@ func (r *TestRunReconciler) reconcile(ctx context.Context, req ctrl.Request, log
 		// delete if configured
 		if k6.GetSpec().Cleanup == "post" {
 			log.Info("Cleaning up all resources")
-			r.Delete(ctx, k6)
+			r.Delete(ctx, k6) //nolint:errcheck
```

Comment:
Can this linter be fixed with smth like:
```
_ = r.Delete(ctx, k6)
```
instead of exception?

Reply:
@yorugac 
We also should be fixed like that below?
```
diff --git a/main.go b/main.go
index 169614b..ed60b2f 100644
--- a/main.go
+++ b/main.go
@@ -89,8 +89,8 @@ func main() {
                os.Exit(1)
        }

-       mgr.AddHealthzCheck("health", healthz.Ping)
-       mgr.AddReadyzCheck("ready", healthz.Ping)
+       mgr.AddHealthzCheck("health", healthz.Ping) //nolint:errcheck
+       mgr.AddReadyzCheck("ready", healthz.Ping)   //nolint:errcheck

        if err = (controllers.NewK6Reconciler(&controllers.TestRunReconciler{
                Client: mgr.GetClient(),
```

Reply:
If the linter allows it, yes please!

Reply:
@yorugac 
I also fixed these.